chia/full_node/full_node_api.py

```diff
@@ -926,13 +928,13 @@ async def request_additions(self, request: wallet_protocol.RequestAdditions) ->
             return msg
 
         assert block is not None and block.foliage_transaction_block is not None
-        _, additions = block.tx_removals_and_additions()
+        additions = await self.full_node.coin_store.get_coins_added_at_height(block.height)
```

Comment:
This will not work in a fork, we need to get the specific additions for this block (which maybe is in a fork)

Reply:
An alternative is to just Reject the request, if we are in a fork